pkg/scalers/temporal.go

```diff
@@ -223,6 +224,9 @@ func getTemporalClient(ctx context.Context, meta *temporalMetadata, log logr.Log
 			},
 		))
 		options.Credentials = sdk.NewAPIKeyStaticCredentials(meta.APIKey)
+		options.ConnectionOptions.TLS = &tls.Config{
+			MinVersion: tls.VersionTLS13,
+		}
```

Comment:
Is this a mandatory parameter for all the connections? I mean, will our test on local will pass?